clang/lib/Basic/Targets/SystemZ.h

```diff
@@ -119,6 +119,12 @@ class LLVM_LIBRARY_VISIBILITY SystemZTargetInfo : public TargetInfo {
                              TargetInfo::ConstraintInfo &info) const override;
 
   std::string convertConstraint(const char *&Constraint) const override {
+    if (llvm::StringRef(Constraint) == "@cc") {
```

Comment:
This should read the same as the check in SystemZ.cpp above.  Why do we have `llvm::StringRef(Constraint) == "@cc"` here and `!StringRef("@cc").compare(Name)` above?  Decide which form is preferable and use that consistently.